app/latigo/metadata_api/metadata_exceptions.py

```diff
@@ -0,0 +1,9 @@
+"""Exceptions for the Metadata API logic."""
+
+
+class MetadataStoringError(Exception):
+    """Raise if got not 200 status code from the API."""
+
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(self.message)
```

Comment:
```suggestion
class MetadataStoringError(Exception):
    """Raise if got not 200 status code from the API."""
```

The added functionality is redundant. Also consider raising existing exception instead of creating a new one.

Reply:
Also could you please point where `MetadataStoringError` is raised?

Reply:
Thanks for noticing, I missed to commit it, it should be [here](https://github.com/equinor/latigo/blob/master/app/latigo/metadata_api/metadata_storage_provider.py#L78(
The main idea why I used custom exceptions cause Latigo time to time raises some unexpected errors and to separate them from known one. And in-build Exceptions are too generic to catch them and ignore on some of the Dashboards.
I tend to leave custom Exceptions cause of the mentioned "surprises" it Latigo, can you offer smth else instead to solve such situation?

P.S. I'll remove the constructor.

Reply:
PR updated